app/jobs/determine_user_role_job.rb

```diff
@@ -26,6 +26,7 @@ def github_jwt
       exp: Time.now.to_i + (10 * 60),
       iss: ENV["GITHUB_APP_ID"],
     }
-    JWT.encode(payload, OpenSSL::PKey::RSA.new(ENV["GITHUB_PRIVATE_KEY"].unpack1("m*"), ''), "RS256")
+    pem = ENV.fetch("GITHUB_PRIVATE_KEY").unpack1("m*") #: String
```

Comment:
I changed `Hash#[]` to `Hash#fetch` because the former one returns String?. Additionally, I added the type comment to indicate the return value of `String#unpack` is ambigous.